var/spack/repos/builtin/packages/py-markdown2/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
```

Comment:
Can you update copyright here and in the other packages:
```suggestion
# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```
?

Reply:
Ok, will do. Thanks!